File: roguskiwarehouse/src/main/java/com/roguskip/roguskiwarehouse/model/ProductView.java
package com.roguskip.roguskiwarehouse.model;

import lombok.Data;

import java.math.BigDecimal;

@Data
public class ProductView {

    private String manufacturerName;
    private String productName;
    private Long manufacturerId;
    private BigDecimal price;
    private Integer quantity;
    private Long productId;
    private String currency;
    private String productGUID;
    private String color;
    private Long warehouseId;
}

File: mobileapp/app/src/main/java/com/roguskip/roguskiwarehouse/model/ProductView.kt
package com.roguskip.roguskiwarehouse.model

import java.io.Serializable
import java.math.BigDecimal


data class ProductView(val manufacturerName : String, val productName : String, var manufacturerId: Int,
                       val price : BigDecimal, val quantity : Int, val productId: Int, val currency: String) : Serializable
